get-docker.sh

```diff
@@ -0,0 +1,630 @@
+#!/bin/sh
```

Comment:
Did you mean to `.gitignore` this file? 🤔

Reply:
Still seeing this file. 🤔

Reply:
I have removed it but testing the image I to discover the size I added it again my mistake

![image](https://user-images.githubusercontent.com/414967/149741444-3f2289eb-b7c2-41b2-8243-dfe91cf61d83.png)